conf.d/30-galera.cnf

```diff
@@ -9,7 +9,7 @@ innodb_lock_schedule_algorithm=FCFS
 
 # Subsequent `wsrep_provider_options` will replace the ones specified below
 wsrep_provider=/usr/lib/galera/libgalera_smm.so
-wsrep_provider_options="gcache.size=2048M; gcache.keep_pages_size=1024M; gcache.recover=yes;"
+wsrep_provider_options="gcache.size=128M; gcache.keep_pages_size=1024M; gcache.recover=yes;"
```

Comment:
Please also fork this file for amd64 vs armv6 because 128M is quite small for a machine without tiny disks so I'd like to keep the default for amd64 the same.